### ISSUE TITLE
Apex icon/logo

### ISSUE THREAD

**Issue by miltoskat** [apex/apex-ui#13]
I saw the lack of an Apex icon for atom, so I went ahead and made one. :)
The download link for the .icns file is [here](https://www.dropbox.com/s/d77qj4j5smzfsj2/apex-icon.icns?dl=0) if you like it add it to your readme so others can download it too.

![icon_128x128 2x](https://cloud.githubusercontent.com/assets/16347487/21732129/ab8e0454-d460-11e6-8b6e-bd699d848539.png)

**Comment by tj** [comment]
nice you can change Atom's icon? didn't know that

**Comment by miltoskat** [comment]
Yeah it's pretty easy to change the Application icons on MacOS you just go on the applications folder, right click the app, go to the get info choice( or you just click the App and then press CMD+I), and you just drag and drop the icns file on the Application icon. and then you run [this script ](https://gist.github.com/miltoskat/51ac60086332b834eae14138d2f3e9af) to refresh the icons cache.

**Comment by tj** [comment]
ahh awesome :D

**Comment by Jarvvski** [comment]
@miltoskat can you re-upload please?
